WorldMoodTracker/js/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import WorldMoodTracker from './components/WorldMoodTracker.js';
 
-export const host = 'http://loklak.org';
+export const host = 'http://35.192.111.68';
```

Comment:
@Achint08: I've set this host because http://loklak.org doesn't have the feature right now as it is behind http://github.com/loklak/loklak_server/tree/development branch and country-wise aggregation was introduced recently.